link/link.go

```diff
@@ -92,11 +99,40 @@ type RawLinkOptions struct {
 	Flags uint32
 }
 
-// RawLinkInfo contains metadata on a link.
-type RawLinkInfo struct {
+// LinkInfo contains metadata on a link.
+type LinkInfo struct {
```

Comment:
Nit: this reads `link.LinkInfo`, `link.Info` is better imo.

Reply:
Renamed.